sceval_frontend/src/containers/Hardware.tsx

```diff
@@ -87,13 +74,13 @@ const Hardware = withRouter((props: HardwareProps & RouteComponentProps) => {
         });
       });
 
-    // Listen to sensorModuleStateChange change event from the web socket and reload the sensor module
+    // Listen to _keyValueSaved_ event from the web socket and reload the sensor module
```

Comment:
I found out that "_keyValueSaved_" event is better to use because it comes with the updated key/value store so we don't need to make an API call to get the latest key/value store.